### PR TITLE
Translate ReactDOM API Reference page

### PR DESCRIPTION
**Pull request by galtalmor**

### CODE REVIEW

File: content/docs/reference-react-dom.md [hunk 1]
```diff
@@ -6,53 +6,55 @@ category: Reference
 permalink: docs/react-dom.html
 ---
 
-If you load React from a `<script>` tag, these top-level APIs are available on the `ReactDOM` global. If you use ES6 with npm, you can write `import ReactDOM from 'react-dom'`. If you use ES5 with npm, you can write `var ReactDOM = require('react-dom')`.
+אם אתם טוענים את React מתג `<script>`, ה-APIs מהרמה העליונה הבאים זמינים תחת המשתתנה הגלובאלי `ReactDOM`. אם אתם משתמשים ב-ES6 עם npm, אתם יכולים לכתוב `import ReactDOM from 'react-dom'`. אם אתם משתמשים ב-ES5 עם npm, תוכלו לכתוב `var ReactDOM = require('react-dom')`.
```
[the remark this answers is not in the document]
Comment (galnir) [reply content/docs/reference-react-dom.md]: And in my opinion גלובאלי should be גלובלי, I looked it up and the correct way to write it is the latter

[the remark this answers is not in the document]
Comment (galtalmor) [reply content/docs/reference-react-dom.md]: Fixed. Thanks.

File: content/docs/reference-react-dom.md [hunk 2]
```diff
@@ -80,28 +82,30 @@ Remember to be mindful of user experience on slow connections. The JavaScript co
...
-> `findDOMNode` is an escape hatch used to access the underlying DOM node. In most cases, use of this escape hatch is discouraged because it pierces the component abstraction. [It has been deprecated in `StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
+> `findDOMNode` היא פתח מילוט המשמשת כדי לגשת לצומת ה-DOM הבסיסי. ברוב המקרים, שימוש בפתח מילוט זה הוא לא מומלץ מכיוון שזה חודר את האבסטרקציה של הקומפוננטה. [היא הוצאה משימוש ב-`StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
+
+
 
 ```javascript
 ReactDOM.findDOMNode(component)
 ```
-If this component has been mounted into the DOM, this returns the corresponding native browser DOM element. This method is useful for reading values out of the DOM, such as form field values and performing DOM measurements. **In most cases, you can attach a ref to the DOM node and avoid using `findDOMNode` at all.**
+אם קומפוננטה זו כבר הופעלה לתוך ה-DOM, קריאה זו מחזירה את אלמנטה ה-DOM התואם המקורי של הדפדפן. מתודה זו שימושית עבור קריאת ערכים מתוך ה-DOM, כגון ערכי שדות טופס וביצוע מדידות DOM. **ברוב המקרים, ניתן לצרף הפנייה לצומת DOM ולהימנע לגמרי משימוש ב-`findDOMNode`.**
```
Comment (galnir) [inline content/docs/reference-react-dom.md]: אלמנטה?

Comment (galtalmor) [reply content/docs/reference-react-dom.md]: Good catch. Thanks!

**Pull request merged**